mass_mailing_custom_unsubscribe/data/mail_unsubscription_reason.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data noupdate="1">
+
+    <record id="reason_not_interested"
+            model="mail.unsubscription.reason"
+            forcecreate="False">
```

Comment:
I think you have to put forcecreate="0", as I'm not sure if this is evaluated as `True` as is not an empty string.